test/fixtures/go-binaries/source/main.go

```diff
@@ -0,0 +1,29 @@
+package main
```

Comment:
this file looks familiar :)
was it copied as is? can use it only once?

Reply:
Copied as is. We don't need the files in `test/go-binary-regression` anymore, so I guess I need to remove those still :)

That brings up another question: should I move this `source` folder to `test/go-binary-regression` instead? Imo keeping it in the `fixtures/go-binaries` directory is clearer as this is the place I'm searching for the source files as well. But YMMV, and I don't have a strong opinion on this :)

Reply:
It makes more sense where it is with changes you made. Just remove the duplicated files from `go-binary-regression`

Reply:
Done.